fhir-swagger-generator/src/main/java/com/ibm/fhir/swagger/generator/FHIRSwaggerGenerator.java

```diff
@@ -259,25 +264,25 @@ private static void generateBatchTransactionSwagger() throws Exception, ClassNot
         swagger.add("paths", paths);
 
 
-        generateDefinition(CapabilityStatement.class, definitions);
-
-        // generate definition for all the defined Types.
-        for (String typeClassName : FHIROpenApiGenerator.getAllTypesList()) {
-            Class<?> typeModelClass = Class.forName(TYPEPACKAGENAME + "." + typeClassName);
-            // System.out.println("Type:  " + className);
-            generateDefinition(typeModelClass, definitions);
-        }
+        generateDefinition(Bundle.class, definitions);
 
         // generate definition for all inner classes inside the top level resources.
         for (String innerClassName : FHIROpenApiGenerator.getAllResourceInnerClasses()) {
             Class<?> parentClass = Class.forName(RESOURCEPACKAGENAME + "." + innerClassName.split("\\$")[0]);
-            if (CapabilityStatement.class.equals(parentClass)) {
+            if (Bundle.class.equals(parentClass)) {
                 Class<?> innerModelClass = Class.forName(RESOURCEPACKAGENAME + "." + innerClassName);
                 // System.out.println("Resource:  " + className);
                 generateDefinition(innerModelClass, definitions);
             }
         }
 
+        // generate definition for all the defined Types.
+        for (String typeClassName : FHIROpenApiGenerator.getAllTypesList()) {
+            Class<?> typeModelClass = Class.forName(TYPEPACKAGENAME + "." + typeClassName);
+            // System.out.println("Type:  " + className);
```

Comment:
Is this commented line for debugging purposes only?

Reply:
If so, maybe add a Boolean DEBUG as switch to turn off by default?

Reply:
i just removed them.  i had removed from FHIROpenApiGenerator before (per your previous PR review), but never removed them from FHIRSwaggerGenerator until now.